### ISSUE TITLE
JVM crash with fasttext

### ISSUE THREAD

**Issue by behrica** [deepjavalibrary/djl#1637]
## Description
Running fasttext on CookingStatckexchange data results in JVM crash.

### Expected Behavior
FastText model gets trained successfully
### Error Message
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGILL (0x4) at pc=0x00007f8d1cc9f72f, pid=10228, tid=10231
#
# JRE version: OpenJDK Runtime Environment (11.0.15+10) (build 11.0.15+10)
# Java VM: OpenJDK 64-Bit Server VM (11.0.15+10, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libjni_fasttext.so+0x9f72f]  fasttext::Args::Args()+0x12f
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h" (or dumping to /home/carsten/Dropbox/sources/try-fasttext-java/core.10228)

```

## How to Reproduce?
Following code shows the issue:
```java
import ai.djl.basicdataset.RawDataset;
import ai.djl.basicdataset.nlp.CookingStackExchange;
import ai.djl.fasttext.FtTrainingConfig;
import ai.djl.fasttext.TrainFastText;

import java.io.IOException;
import java.nio.file.Path;

public class TryFT {
    public static void main(String[] args) throws IOException {
        FtTrainingConfig ftTrainingConfig = FtTrainingConfig.builder().
                setModelName("my").setOutputDir(Path.of("/tmp")).
                build();
        RawDataset<Path> rawDataset = CookingStackExchange.builder().build();
        TrainFastText.textClassification(ftTrainingConfig, rawDataset);
    }
}

```
### Steps to reproduce
(Paste the commands you ran that produced the error.)

1. Run above code

## What have you tried to solve it?

1. Looked for similar issues like #1535 , but not sure if the same

## Environment Info
djl 0.16.0 on Arch Linux with Java11

**Comment by frankfliu** [comment]
@behrica 

Thanks for reporting this issue. We noticed there is random crash for fastText, will look into this bug.

For the time being, you can try DJL 0.15.0

**Comment by behrica** [comment]
I did not have it in 0.15.0, so I believe as well that it is new. 

Thanks for investigating.

**Comment by behrica** [comment]
solved in DJL 0.17.0
